src/services/useRequestRepoSearchAPI.tsx

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+import { useState } from "react";
```

Comment:
Always put the react import to the top then other 3rd party libs, then your components, your helper functions, your types and then your styles.